src/inc/regdisp.h

```diff
@@ -11,7 +11,7 @@ class Thread;
 #endif // DEBUG_REGDISPLAY
 
 
-#if defined(_TARGET_X86_)
+#if defined(_TARGET_X86_) && !defined(WIN64EXCEPTIONS)
```

Comment:
I agree that code cleanup need to be done separately. I created #8643 as a staring point of this cleanup issue.